core/src/main/java/com/linecorp/armeria/common/MediaType.java

```diff
@@ -422,7 +422,7 @@ private static MediaType addKnownType(MediaType mediaType) {
     public static final MediaType PDF = createConstant(APPLICATION_TYPE, "pdf");
     public static final MediaType POSTSCRIPT = createConstant(APPLICATION_TYPE, "postscript");
     /**
-     * <a href="http://tools.ietf.org/html/draft-rfernando-protocol-buffers-00">Protocol buffers</a>
+     * <a href="http://tools.ietf.org/html/draft-rfernando-protocol-buffers-00">Protocol buffers</a>.
```

Comment:
I wish I can but IIRC @hyangtack made some changes on top of the fork so it's not a pure fork anymore unfortunately.